public/app/features/trails/autoQuery/queryGenerators/histogram.ts

```diff
@@ -0,0 +1,78 @@
+import { VAR_GROUP_BY_EXP, VAR_METRIC_EXPR } from '../../shared';
+import { heatmapGraphBuilder, percentilesGraphBuilder, simpleGraphBuilder } from '../graphBuilders';
+import { AutoQueryContext, AutoQueryDef } from '../types';
+
+import { generateBaseQuery } from './baseQuery';
+
+export function createHistogramMetricQueryDefs(context: AutoQueryContext) {
+  const { unit } = context;
+  const title = `${VAR_METRIC_EXPR}`;
```

Comment:
I'm curious about the purpose of this, as opposed to just using `VAR_METRIC_EXPR` directly

Reply:
I tested what you suggest and it just change nothing :)

Reply:
Here is the cleaned version https://github.com/grafana/grafana/pull/97815/commits/006709fd10434dbfdbef643b96615a69b77f5d56